src/features/Sync/SyncOutManager.ts

```diff
@@ -0,0 +1,94 @@
+import { ENTITIES } from "../../common/constants";
+import { SyncLog } from "../../common/services/Database/entities";
+
+const HOSTNAME = '10.0.2.2';
+const PORT = '8080';
+
+const RESOURCES = {
+    COLDCHAIN_V1: 'coldchain/v1'
+}
+
+const PATHS = {
+    LOGIN: 'login',
+    SENSOR: 'sensor', 
+    TEMPERATURE_LOG: 'temperature-log',
+    TEMPERATURE_BREACH: 'temperature-breach'
+}
+class SyncOutManager {
+    private paths: { [key: string]: string };
+
+    constructor() {
+        // TODO: initialise SyncOut manager from dependency container.
+        const rootPath = `http://${HOSTNAME}:${PORT}/${RESOURCES.COLDCHAIN_V1}`;
+
+        const loginPath = `${rootPath}/${PATHS.LOGIN}`;
+        const sensorPath = `${rootPath}/${PATHS.SENSOR}`;
+        const temperatureLogPath = `${rootPath}/${PATHS.TEMPERATURE_LOG}`;
+        const temperatureBreachPath = `${rootPath}/${PATHS.TEMPERATURE_BREACH}`;
+
+        this.paths = {
+            login: loginPath,
+            sensor: sensorPath,
+            temperatureLog: temperatureLogPath,
+            temperatureBreach: temperatureBreachPath,
+        };
+    }
+
+    
+
+    pushLogs = async(logs: SyncLog[]) => {
+        const loginResponse = await fetch(this.paths.login, {
```

Comment:
This is all very coupled at the moment. 

Due to having multiple endpoints, batching is a bit more complicated than in mobile. E.g. just taking a batch of `n` records could lead to pushing `~n` records to one endpoint (probably `temperature-log`), and only a few to the other endpoints). Not quite sure how to resolve this, but best idea is probably to group records by type before batching them?